packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -140,6 +142,22 @@ export const HvDialog = ({
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
+  const slots = useMemo<ModalSlots>(
+    () => ({
+      backdrop: (backdropProps) => (
+        <StyledBackdrop
+          open={open}
+          onClick={(event) => wrappedClose(event)}
+          $backColor={
+            activeTheme?.colors?.modes[selectedMode].atmo4 || theme.colors.atmo4
+          }
+          {...backdropProps}
+        />
+      ),
+    }),
+    []
```

Comment:
Missing the dependency array! Without it we might get wrong behaviour when `open`, `backdropProps`, `activeTheme`, `onClose` change.

Reply:
Is there another way to maintain the state of the component and make `StyledBackdrop` fixed?

Reply:
Resolved on my last update 🚀